test/server-test.js

```diff
@@ -0,0 +1,38 @@
+const assert  = require('chai').assert
+const app     = require('../server')
+const request = require('request')
+
+describe('Server', () => {
+  before( (done) => {
+    this.port = 9001
+
+    this.server = app.listen(this.port, (error, result) => {
+      if (error) { return done(error) }
+      done()
+    })
+
+    this.request = request.defaults({
+      baseUrl: 'http://localhost:9001/'
+    })
+  })
+
+  after( () => {
```

Comment:
When the callback is the only argument, you can skip the space between the first two open parens `after(() => {`